### PR TITLE
feat: re-add homebrew bump flow

### PR DESCRIPTION
**Pull request by chenrui333**
re-add homebrew bump flow per [formula rename](https://github.com/Homebrew/homebrew-core/pull/196439).

---

using official brew actions this time so that the npmjs artifact issue would be resolved see [here](https://github.com/Homebrew/homebrew-core/pull/196454) and [here](https://github.com/Homebrew/homebrew-core/pull/196430)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: release
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  homebrew:
+    runs-on: macos-latest
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply .github/workflows/release.yml:10]: I think it is fine.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+      - name: Set up Homebrew
+        uses: Homebrew/actions/setup-homebrew@master
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply .github/workflows/release.yml:13]: yeah, we probably like to use master

**Comment by yamadashy** [comment]
Thank you for the PR!
I was thinking about re-adding the previous automation, but moving to official Homebrew actions is definitely an improvement.

I've checked the source of actions, and this new approach is much more robust and trustworthy.

**Review by yamadashy — APPROVED** [review]

**Comment by yamadashy** [comment]
@chenrui333 
Thank you so much for improving our Homebrew automation!

Feel free to open more PRs if you find anything else that could be enhanced!

**Comment by chenrui333** [comment]
yeah, totally, thanks for the support!

**Pull request merged**